### DISCUSSION TITLE
Any suggestions on how to copy files into /out from vm

### DISCUSSION THREAD

**Discussion by bit-lang** [copy/v86#1156] [Q&A]
Is there a standard means to copy or import export files between guest and API?

**Comment by bit-lang** [comment]
there is a brief intro here: https://github.com/copy/v86/blob/master/docs/filesystem.md
could anyone elaborate so that I can follow the instructions?

**Comment by copy** [comment]
For getting files into the VM, you can now use the cdrom: https://github.com/copy/v86/issues/54#issuecomment-2965689561

**Reply by bit-lang** [reply]
> For getting files into the VM, you can now use the cdrom: [#54 (comment)](https://github.com/copy/v86/issues/54#issuecomment-2965689561)

this is awesome 

thank you for the great improvements
